tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+skipsdist = true
+envlist = py{27,35}
+
+[tox:travis]
+2.7 = py27
+3.4 = py34
+
+[testenv]
+passenv = DISPLAY
+deps =
+    pytest==3.0.3
+    pytest-selenium==1.4.0
+    selenium==3.0.1
+commands = pytest --driver Firefox {posargs}
+
+[pytest]
+testpaths = foxpuppet/tests
```

Comment:
Tests should not be included in the `foxpuppet` package - please move them to the top level.